sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/ReadConsistencyStrategy.java

```diff
@@ -0,0 +1,103 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.cosmos;
+
+import com.azure.cosmos.implementation.ImplementationBridgeHelpers;
+import com.azure.cosmos.util.Beta;
+import com.fasterxml.jackson.annotation.JsonValue;
+
+import java.util.HashMap;
+import java.util.Map;
+
+/**
+ * Represents the read consistency strategies supported by the Azure Cosmos DB service.
+ * <p>
+ * The requested read consistency strategy can be chosen independent of the consistency level
+ * provisioned for the database account.
+ * <p>
+ * NOTE: The ReadConsistencyStrategy is currently only working when using direct mode
+ */
+@Beta(value = Beta.SinceVersion.V4_69_0, warningText = Beta.PREVIEW_SUBJECT_TO_CHANGE_WARNING)
+public enum ReadConsistencyStrategy {
+
+    /**
+     * Use the default read behavior for the consistency level applied to the operation, the client or the account
+     */
+    DEFAULT("Default"),
+
+    /**
+     * Eventual Consistency guarantees that reads will return a subset of writes. All writes
+     * will be eventually be available for reads.
+     */
+    EVENTUAL("Eventual"),
+
+    /**
+     * Session Consistency guarantees monotonic reads (you never read old data, then new, then old again), monotonic
+     * writes (writes are ordered) and read your writes (your writes are immediately visible to your reads) within
+     * any single session.
+     */
+    SESSION("Session"),
+
+    /**
+     * Will read the latest committed version from the region in preferred order (which means the read region might
+     * have stale data) but this read strategy will return the latest committed version of that region
+     */
+    LATEST_COMMITTED("LatestCommitted"),
+
+    /**
+     * Will do a quorum read followed by GCLSN barrier requests if needed.
```

Comment:
Do we modify this to something more generic / customer friendly? It is like `LATEST_COMMITTED` but also global.

Reply:
Fixed